pkg/controller/command/command_config_bootstrap.go

```diff
@@ -266,3 +268,48 @@ resources:
       public_url: https://{{ .SwiftProxyAddress }}:{{ .SwiftProxyPort }}
     kind: endpoint
 `))
+
+var migrationConfig = template.Must(template.New("").Parse(`
+database:
+  host: {{ .PostgresAddress }}
+  user: {{ .PostgresUser }}
+  password: {{ .PGPassword }}
+  name: {{ .PostgresDBName }}_migrated
+  max_open_conn: 100
+  connection_retries: 10
+  retry_period: 3s
+  replication_status_timeout: 10s
+  debug: false
+
+log_level: debug`))
+
+var migrationScriptConfig = template.Must(template.New("").Parse(`
+#!/usr/bin/env bash
+
+set -e
+export PGPASSWORD={{ .PGPassword }}
+
+# Try to drop old databases the may or may not exists.
+psql -w -h {{ .PostgresAddress }} -U {{ .PostgresUser }} -d {{ .PostgresDBName }} <<END_OF_SQL
```

Comment:
[nit] just a nit if you are exporting PGPASSWORD maybe it will be convenient  to store PostgresAddress and PostgresUser in env too: https://www.postgresql.org/docs/9.3/libpq-envars.html

Reply:
changed